examples/advanced/03_web_frameworks/django_project/chat/views.py

```
@@ -0,0 +1,286 @@
+"""
+Django 뷰
+"""
+import json
+from django.http import JsonResponse, StreamingHttpResponse
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+from django.core.cache import cache
+from django.contrib.auth.decorators import login_required
```

Ruff (0.11.9)

`django.contrib.auth.decorators.login_required` imported but unused

Remove unused import: `django.contrib.auth.decorators.login_required`

(F401)